Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -768,6 +791,7 @@ private void RemoveBucket(GcsPath gcsPath, bool removeObjects)
             if (removeObjects)
             {
                 DeleteObjects(gcsPath);
+                Thread.Sleep(100);
```

Comment:
Like before, is this absolutely necessary? Was adding a delay here necessary to prevent errors when deleting the bucket, e.g. it would throw a 409? If not, let's remove it. (Or, catch the 409 exception and try again once more.)

Static sleeps are a code smell.